### PR TITLE
Lw add dev configs

### PR DESCRIPTION
**Pull request by lindawoo-plaid**
This PR removes account-funding functionality and instead adds the ability for user to use TransferUI to make an initial subscription payment. 
1) adds transfer endpoints for creation of transfer_intent_id and to get status of transfer
2) adds transfer_intent_id to link_token_creation params in order to use TransferUI
3) adds transfers table and queries to the database and removes account-funding data from accounts table
4) adds TransfersProvider for context
5) adds some placeholder UI so I can test the endpoints

### CODE REVIEW

File: server/routes/linkTokens.js [hunk 1]
```diff
@@ -50,13 +55,25 @@ router.post(
...
+        },
+        link_customization_name: 'transfer_ui_woo',
       };
 
       // If user has entered a redirect uri in the .env file
       if (redirect_uri.indexOf('http') === 0) {
         linkTokenParams.redirect_uri = redirect_uri;
       }
-      const createResponse = await plaid.linkTokenCreate(linkTokenParams);
+      const createResponse = await axios.post(
```
Comment (mattnguyen1) [inline server/routes/linkTokens.js:68]: any reason we can't continue using the plaid client here?

File: server/routes/transfers.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        amount: subscriptionAmount.toFixed(2),
+        ach_class: 'ppd',
+        description: 'foobar',
+        user: {
+          legal_name: 'Linda Woo',
+        },
+      };
+      let transferIntentId;
+
+      const transferIntentCreateResponse = await axios.post(
```
Comment (mattnguyen1) [inline server/routes/transfers.js:43]: similar to the above, is there any reason we can't use the plaid client for this file?

**Review by mattnguyen1 — COMMENTED** [review]
nice job so far! left a few comments

File: client/src/components/Item.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ const Item: React.FC<Props> = (props: Props) => {
   const initiateLink = async () => {
     // only generate a link token upon a click from enduser to add a bank;
     // if done earlier, it may expire before enuser actually activates Link to add a bank.
-    await generateLinkToken(props.userId, null, isIdentity);
+    await generateLinkToken(props.userId, null, 'lkjlj'); // will fix this later
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply client/src/components/Item.tsx:47]: yes, for now...

File: client/src/components/LinkButton.tsx [hunk 4]
```diff
@@ -11,39 +11,47 @@ import {
...
 interface Props {
   isOauth?: boolean;
   token: string;
   userId: number;
   itemId?: number | null;
   children?: React.ReactNode;
-  isProcessor: boolean;
-  isIdentity: boolean;
+  isProcessor?: boolean; // will remove later
+  isIdentity?: boolean; // will remove later
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply client/src/components/LinkButton.tsx:30]: not yet...

File: client/src/components/LinkButton.tsx [hunk 5]
```diff
@@ -52,11 +60,44 @@ const LinkButton: React.FC<Props> = (props: Props) => {
...
+            sweep_status,
+            data.items[0].id
+          );
+        }
+
+        await getTransfersByUser(props.userId);
+
+        // const transferData = await getTransferStatus(
+        //   transferUIData.data.transfer_intent
+        // );
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply client/src/components/LinkButton.tsx:100]: deleted

File: client/src/components/LinkButton.tsx [hunk 6]
```diff
@@ -52,11 +60,44 @@ const LinkButton: React.FC<Props> = (props: Props) => {
...
+          } = transferDataResponse.data.transfer;
+          // update database with information regarding the transfer
+          await addTransferInfo(
+            transfersData.transfer_intent_id,
+            account_id,
+            id,
+            origination_account_id,
+            status,
+            sweep_status,
+            data.items[0].id
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply client/src/components/LinkButton.tsx:92]: yes, it is

File: server/routes/institutions.js [hunk 7]
```diff
@@ -80,7 +80,7 @@ router.get(
           url,
         },
       };
-      prettyPrintResponse(responseData);
+      // prettyPrintResponse(responseData);
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply server/routes/institutions.js:83]: i uncommented this for testing purposes...it was getting really annoying each time to see all this data

File: server/routes/linkTokens.js [hunk 8]
```diff
@@ -24,14 +27,14 @@ router.post(
...
       let accessToken = null;
-      let products = ['auth']; // must include transactions in order to receive transactions webhooks
-      if (isIdentity) {
-        products.push('identity');
-      }
+      let products = ['transfer'];
+
       if (itemId != null) {
         // for the link update mode, include access token and an empty products array
+        // ask Greg Manley what to do with TransferUI and update mode...do i remove the transfer
```
[the remark this answers is not in the document]
Comment (lindawoo-plaid) [reply server/routes/linkTokens.js:36]: okay, i will remove

File: server/routes/linkTokens.js (hunk 1 above)
Comment (lindawoo-plaid) [reply server/routes/linkTokens.js:68]: does not support transferUI!!!

File: server/routes/transfers.js (hunk 2 above)
Comment (lindawoo-plaid) [reply server/routes/transfers.js:43]: none yet for transfers

**Pull request merged**